example/lib/screen/home_page.dart

```diff
@@ -101,19 +128,140 @@ class Homepage extends StatelessWidget {
               );
             },
             child: const Text("Footer"),
-          ), ElevatedButton(
-            onPressed: () {
-              Navigator.push(
-                context,
-                MaterialPageRoute(
-                  builder: (context) => const GridViewScreen(),
-                ),
-              );
-            },
-            child: const Text("Gridview with Hover"),
           ),
         ],
       ),
     );
   }
+  Future<void> _createTutorial() async {
```

Comment:
write a widget function to deconstruct it into smaller widget and pass the required data